### DISCUSSION TITLE
请问作者开发的时候是如何快速 debug 源码的呢？

### DISCUSSION THREAD

**Discussion by liuweiGL** [alibaba/formily#1214] [Q&A 2.x] [answered]
## 问题
1. 我把 2.0 的源码拉取下面，按照文档把依赖都安装好了。
2. 我在 `packages/reactive/src` 下面建个 `demo.ts`:

![image](https://user-images.githubusercontent.com/21048878/114512512-c6762f80-9c6b-11eb-8742-926d5f1f7586.png)

3. 如何直接 debug `demo.ts` 呢？

---

这个问题跟项目无关，纯粹是向作者讨教，十分感谢。

**Comment by janryWang** [comment, marked as answer]
大多数情况我是在单元测试里调试问题的，如果涉及UI，那就用dumi文档

**Comment by liuweiGL** [comment]
好的，我试试。
